p2p_wallet/Scenes/Main/InvestSolend/InvestSolendView.swift

```diff
@@ -0,0 +1,116 @@
+// Copyright 2022 P2P Validator Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style license that can be
+// found in the LICENSE file.
+
+import KeyAppUI
+import Resolver
+import SolanaSwift
+import Solend
+import SwiftUI
+
+struct InvestSolendView: View {
+    @StateObject var viewModel: InvestSolendViewModel
+    
+    var body: some View {
+        VStack {
+            // Title
+            HStack {
+                Text(L10n.earnAYield)
+                    .fontWeight(.bold)
+                    .apply(style: .largeTitle)
+                Spacer()
+            }
+            .padding(.top, 20)
+            .padding(.horizontal, 16)
```

Comment:
Don't use this, use navigationTitle instead this. It's a dirty hook

Reply:
We've tried but couldn't make navigationTitle to adapt the design.

Reply:
With navigationTitle and largeTitle = true, the title is correct, but the button will be at the top, not inline with the title. So we couldn't use navigationTitle

Reply:
Design:
![image](https://user-images.githubusercontent.com/6975538/192478751-7a578493-3dbd-4ff9-a0d1-098ac0b0af09.png)
with navigationTitle and "rightBarButton":
![image](https://user-images.githubusercontent.com/6975538/192479076-60b05eee-358a-408f-998f-5af49759f3ee.png)

Reply:
There is a solution for this, but it has the risk that application might be rejected by Apple for using private api
https://stackoverflow.com/a/63748032

Reply:
If you have any ideas, please help us by opening PR. We're appreciated!